.github/workflows/test.yaml

```diff
@@ -0,0 +1,44 @@
+name: Tests
+on:
+  - push:
+  - pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - "3.8"
```

Comment:
Added this here just because I'm planning to work on python 3.9 support next. I can also not have the matrix if preferred.